tests/conftest.py

```diff
@@ -109,9 +109,41 @@ def DSHAPE(tmpdir_factory):
     return DSHAPE_out
 
 
+@pytest.fixture(scope="session")
+def DSHAPE_current(tmpdir_factory):
+    """Run DSHAPE fixed toroidal current example."""
+    input_path = ".//tests//inputs//DSHAPE_current"
+    output_dir = tmpdir_factory.mktemp("result")
+    desc_h5_path = output_dir.join("DSHAPE_current_out.h5")
+    desc_nc_path = output_dir.join("DSHAPE_current_out.nc")
+    # TODO: change these paths too? (note that DSHAPE_current converges to DSHAPE)
```

Comment:
this is probably fine, if DSHAPE and DSHAPE current converge to same thing don't need to add one more netcdf